src/haz3lweb/Main.re

```diff
@@ -48,7 +31,7 @@ let apply = (model, action, state, ~schedule_action): Model.t => {
   last_edit_action := JsUtil.timestamp();
   switch (
     try({
-      let new_model = Update.apply(model, action, state, ~schedule_action);
+      let new_model = Update.apply(model, action, (), ~schedule_action);
```

Comment:
Should we just remove state from `Update.apply`?

Reply:
yeah why not